runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -68,6 +52,29 @@ object GrpcClientSettings {
     GrpcClientSettings.fromConfig(clientConfig)
   }
 
+  /**
+   * Configure the client to lookup a valid hostname:port from a service registry accessed via the Service
+   * Discovery implementation provided. Other than the provided parameters, this factory method will read all other
+   * values from `application.conf`.
+   *
+   * @param serviceName name of the remote service to lookup.
+   * @param discovery   service discovery instance to resolve the lookup.
+   */
+  def connectTo(serviceName: String, discovery: SimpleServiceDiscovery)(implicit actorSystem: ActorSystem): GrpcClientSettings = {
+    val clientConfiguration: Config = actorSystem.settings.config.getConfig("akka.grpc.client").getConfig("\"*\"")
+    val resolveTimeout = clientConfiguration.getDuration("service-discovery.resolve-timeout").asScala
+    new GrpcClientSettings(serviceName, discovery, -1, resolveTimeout)
+      .copy(
+        servicePortName = getOptionalString(clientConfiguration, "service-discovery.port-name"),
```

Comment:
It would be good if we only needed those magic strings in one place... could we refactor this so we could do something like

```
fromConfig("*")
  .withDiscovery(discovery)
```
Currently `fromConfig` validates that there is a discovery mechanism defined, that would have to be factored out, but that seems doable. WDYT?